src/components/views/Causes.js

```diff
@@ -15,9 +15,8 @@ class Causes extends Component {
 
   removeCause(id){
     console.log('click', id);
-    socket.emit("causes::remove", id, { cascade: true }, (error, cause) => {
-      console.log('Remove a cause', cause);
-    })
+    const causes = feathersClient.service('/causes');
+    causes.remove(id).then(cause => console.log('Remove a cause', cause));
```

Comment:
This method removes from server and emits event.  Don't fully understand but found this in docs: "every service will automatically become an instance of a NodeJS EventEmitter." (found in https://docs.feathersjs.com/api/services#service-methods and https://docs.feathersjs.com/api/services#feathers-functionality)